### ISSUE TITLE
Add option to show QuickFolders tabs below main toolbar

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#20]
Add a new option that allows displaying the QF tabs closer to the preview area (below the main toolbar) like this:

![image](https://user-images.githubusercontent.com/6314674/72089420-ebab1e00-3303-11ea-87db-a19fa55dde47.png)

**Comment by PissedFirefoxUser** [comment]
Sure hope you can make this work! Thanks for all your excellent add-ons!

**Comment by RealRaven2000** [comment]
In Inspector, with MenuOnTop installed, I can move the toolbar to the desired location with this:

```
#QuickFolders-Toolbar {
  -moz-box-ordinal-group: 40 !important;
}
```

I think the main thing here is to give it a higher value than the main toolbar `#mail-bar3`

This method should also work as a rule in `userChrome.css`.

**Comment by RealRaven2000** [comment]
Upon testing without MenuOnTop, the same rule should also work in this case as by default there is no `-moz-box-ordinal-group` set for `#mail-bar3` - so the value counts as 0. Only MenuOnTop adds the value 30 (and maybe other Add-ons that may influence toolbar location order).

This means a high value for `-moz-box-ordinal-group` will work in any case and push down the QF toolbar to the bottom of the container `#mail-toolbox`. I think 100 should be fine.

**Comment by RealRaven2000** [comment]
As regards configuration UI, the Options dialog is already fairly overloaded - is it okay to hide the configuration behind the General item "Hide QuickFolders when they are not needed"? This could just open a small config window (a method I have used before for other deep configuration to unclutter QF's UI) where advanced toolbar options can be modified.

![image](https://user-images.githubusercontent.com/6314674/72142607-f65dc580-338c-11ea-93c0-d90a09a7daf6.png)

apart from removing clutter, I also won't have to add multi-language translation for this feature.

**Comment by RealRaven2000** [comment]
Below is a test version. To set the new option do the following steps

- open QuickFolders settings, General tab
- right-click the option _Hide QuickFolders when they are not needed_
- find the setting `extensions.quickfolders.toolbar.ordinalPosition` and set it to 40 or a higher value.
- close the configuration window

![image](https://user-images.githubusercontent.com/6314674/72143785-463d8c00-338f-11ea-900d-d6a9142df855.png)

To set to normal, right-click the value to reset to its default value ( 0 ).

![image](https://user-images.githubusercontent.com/6314674/72144005-a7fdf600-338f-11ea-9fc4-4227bd3035c5.png)

[QuickFolders-min-4.18pre3.zip](https://github.com/RealRaven2000/QuickFolders/files/4045002/QuickFolders-min-4.18pre3.zip)

**Comment by PissedFirefoxUser** [comment]
It worked perfectly!! Thank you!

**Comment by RealRaven2000** [comment]
Fixed in release 4.18.1 - published today (07/06/2020). Setting can be found on the _Advanced_ tab of QuickFolders Settings.

![image](https://user-images.githubusercontent.com/6314674/83966470-4ac08f00-a8b2-11ea-9629-e8aad369fdaa.png)
